doc/user/content/sql/execute.md

```diff
@@ -22,10 +22,18 @@ Field | Use
 
 ## Example
 
+The following example [prepares a statement](/sql/prepare/) `a` and runs it
```

Comment:
Later on (when I update the syntax to remove the rr diagram), I'll rename `a` to something else and will mention what it actually does.